bigchaindb/common/transaction.py

```diff
@@ -421,7 +420,7 @@ class Transaction(object):
     TRANSFER = 'TRANSFER'
     GENESIS = 'GENESIS'
     ALLOWED_OPERATIONS = (CREATE, TRANSFER, GENESIS)
-    VERSION = '.'.join(bigchaindb.version.__short_version__.split('.')[:2])
+    VERSION = '1.0'
```

Comment:
Tim made a TODO list of all the things that must be done, including updating all the drivers: https://github.com/bigchaindb/bigchaindb/issues/1558#issuecomment-310317072